### ISSUE TITLE
[DOM] Input elements should have autocomplete attributes (suggested: "current-password"): (More info: https://goo.gl/9p2vKq) <input aria-invalid=​"false" id=​":​r3q:​" required type=​"password" class=​"MuiInputBase-input MuiOutlinedInput-input css-1x5jdmq" value>​

### ISSUE THREAD

**Issue by neg-0** [neg-0/dod-fitness#8]
[DOM] Input elements should have autocomplete attributes (suggested: "current-password"): (More info: https://goo.gl/9p2vKq) <input aria-invalid=​"false" id=​":​r3q:​" required type=​"password" class=​"MuiInputBase-input MuiOutlinedInput-input css-1x5jdmq" value>​
